src/domain/community/application/application.entity.ts

```diff
@@ -19,6 +19,12 @@ export class Application extends AuthorizableEntity implements IApplication {
   @Column()
   ecoverseID?: string;
 
+  @Column()
+  createdDate!: Date;
+
+  @Column()
+  updatedDate!: Date;
+
```

Comment:
Is there a way to put this in the base entity?

Reply:
@techsmyth, should I?

Reply:
It is already there in the base entity. 
Application extends AuthorizableEntity 
AuthorizableEntity extends BaseAlkemioEntity:

![image](https://user-images.githubusercontent.com/30729240/135651934-e523629c-e8c2-4311-84a6-30db28380a1c.png)